src/main/java/uk/nhs/hee/tis/trainee/sync/service/DbcSyncService.java

```diff
@@ -99,23 +99,46 @@ public Optional<Dbc> findByDbc(String dbc) {
     return repository.findByDbc(dbc);
   }
 
+  public Optional<Dbc> findByAbbr(String abbr) {
+    return repository.findByAbbr(abbr);
+  }
+
   /**
    * Make a request to retrieve a specific Dbc.
    *
    * @param dbc The designated body code of the Dbc to be retrieved.
    */
-  public void request(String dbc) {
-    if (!requestCacheService.isItemInCache(Dbc.ENTITY_NAME, dbc)) {
-      log.info("Sending request for DBC [{}]", dbc);
+  public void requestByDbc(String dbc) {
+    request("dbc", dbc);
+  }
+
+  /**
+   * Make a request to retrieve a specific Dbc.
+   *
+   * @param abbr The designated body abbreviation of the Dbc to be retrieved.
+   */
+  public void requestByAbbr(String abbr) {
+    request("abbr", abbr);
+  }
+
+  /**
+   * Make a request to retrieve a specific Dbc.
+   *
+   * @param key   The field to filter the Dbc records by.
+   * @param value The field value of the Dbc to be retrieved.
+   */
+  private void request(String key, String value) {
+    if (!requestCacheService.isItemInCache(Dbc.ENTITY_NAME, value)) {
```

Comment:
'value' can be either a dbc or an abbr. These should never overlap (share the same value), so there should never be key collisions - but if desired we could either prefix them with the type of key (to ensure uniqueness) or have two separate caches.